package.json

```diff
@@ -4,7 +4,7 @@
     "email": "info@sussol.net"
   },
   "name": "mSupplyMobile",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
@edmofro @nmadruga @andreievg It's a bit late now but in development it'd have been better to use 2.0.0-rc(what ever number) or beta tag. I think it'd be tidier to keep the next release as 2.0.0 (which we can still do as we haven't released these testing versions).